packages/shared/lib/core/nfts/utils/getIpfsHash.ts

```diff
@@ -0,0 +1,7 @@
+export function getIPFSHash(url?: string): string | undefined {
+    const ipfsPrefix = 'ipfs'
+
+    if (url?.includes(ipfsPrefix)) {
+        return url.slice(ipfsPrefix.length)
```

Comment:
This method now returns url with first 4 characters sliced ex. `s://ipfs.io/ipfs.......` instead of ipfs hash.